about.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="about-styles.css">
+</head>
+<body>
+
+  <section class="about">
+    <h1 class="about-title">About me</h1>
+    <p class="about-me">Hello I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p class="resume-link"><a href="#">Get My Resume</a></p>
```

Comment:
- [ ] The button `See project` is supposed to be a link `a` or `button` tag for semantic reasons, you don't have to grab it in a paragraph since it's not a text paragraph, and follow the button styling: `Active`, `Hover` and `Pressed` as shown on the design in Figma